.env.alfajores

```diff
@@ -54,7 +54,7 @@ GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-org/geth"
 GETH_NODE_DOCKER_IMAGE_TAG="1.3.0"
 
 GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth-all"
-GETH_BOOTNODE_DOCKER_IMAGE_TAG="master"
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="e7acdda7d2f814c8068e303d5034f5f9f662a1b8"
```

Comment:
Is there a reason to use the commit hash instead of "1.3.0" here (and similarly in the others)?